src/main/webapp/app/entities/programming-exercise/programming-exercise-update.component.ts

```diff
@@ -137,4 +128,22 @@ export class ProgrammingExerciseUpdateComponent implements OnInit {
     trackCourseById(index: number, item: Course) {
         return item.id;
     }
+
+    onNewProgrammingLanguage() {
+        if (!this.startedEditing) {
```

Comment:
I'm not sure if this  is a good idea - this would mean it is not possible to reload the template of another programming language if the instructions were edited. Users sometimes then might not know that there even is a different template for a different language.
This should be implemented with a save guard, as e.g. in the `code-editor-mode-container.component`.